chat-plugins/github.js

```diff
@@ -0,0 +1,129 @@
+/**
+ * GitHub Alert Notifications
+ * Pokemon Showdown - http://pokemonshowdown.com/
+ *
+ * This notifies a set of rooms of changes to specific repositor(y/ies).
+ *
+ * Much of this comes from Ecuacion's bot, which is based on xfix's bot,
+ * and jd who originally converted this to PS server-side.
+ *
+ * @license MIT license
+ */
+
+'use strict';
+
+let gitConfig = {};
+
+if (!Config.github) {
+	return;
+} else if (Config.github === {}) {
```

Comment:
`Config.github` and `{}` will never be equal. Try comparing their return values for `Object.keys` or something instead